x-pack/plugins/file_upload/public/importer/geo/geojson_importer/geojson_importer.ts

```diff
@@ -14,8 +14,15 @@ import { AbstractGeoFileImporter } from '../abstract_geo_file_importer';
 
 export const GEOJSON_FILE_TYPES = ['.json', '.geojson'];
 
+interface LoaderBatch {
+  bytesUsed?: number;
+  batchType?: string;
+  container?: Feature;
+  data?: Feature[];
+}
```

Comment:
This interface is just assumed based on the way that the `batch` variable was used, likely not complete but seems good enough for what we're doing here.